tests/torchtune/modules/test_attention_utils.py

```diff
@@ -82,10 +82,6 @@ def test_packed_block_causal_mask_sdpa(self, seq_lens):
         )
         torch.testing.assert_close(actual, expected)
 
-    @pytest.mark.skipif(
```

Comment:
Ok looks like we need to keep this check, in case the hardware that runs the gpu tests on GitHub CI does not support flex attention